### PR TITLE
Add S3 bucket for storing recipes json dumps

### PR DESCRIPTION
**Pull request by michel-ds**
For people who prefer working with a flat file during the hack, this bucket can host those files.

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: lib/recipe-data-hackday-2023.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ export class RecipeDataHackday2023 extends GuStack {
 			sortKey: { name: 'recipeId', type: AttributeType.NUMBER },
 		});
 
+		new s3.Bucket(this, 'StructuredRecipes');
```
Comment (akash1810) [inline lib/recipe-data-hackday-2023.ts]: Might be worth setting the [`bucketName` prop](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.Bucket.html#bucketname), as by default the bucket name will have a random GUID, which isn't the easiest to remember in a hack day.

Comment (michel-ds) [reply lib/recipe-data-hackday-2023.ts]: Good point! Added in https://github.com/guardian/recipe-data-hackday-2023/pull/1/commits/48aecf610d19f3f283d22a530902e55ea334f272

**Comment by michel-ds** [comment]
@akash1810 would you be able to generate a CF snapshot with the bucket in, please?
Not quite sure how to do that. That's the reason why the unit test is failing atm.

**Comment by akash1810** [comment]
> generate a CF snapshot with the bucket in, please?
> Not quite sure how to do that.

[`npm run test-update`](https://github.com/guardian/recipe-data-hackday-2023/blob/017bb63eeb473edac17d99d039dc17825b945fc5/package.json#L8) should do it.

**Comment by michel-ds** [comment]
I assume this [credentials error](https://github.com/guardian/recipe-data-hackday-2023/actions/runs/4959377449) is because the repo was forked?
Unfortunately, I don't have the rights to work of a branch in the main repo. :(

**Comment by akash1810** [comment]
> Unfortunately, I don't have the rights to work of a branch in the main repo.

Ah, I always forget which group you're in. I've given [`all`](https://github.com/orgs/guardian/teams/all) write permissions, on the assumption that everyone is in there...

For ease, I'll re-run the build, which should resolve the credentials issue you're seeing.

**Review by akash1810 — APPROVED** [review]

**Comment by michel-ds** [comment]
Brill, thank you!

**Comment by akash1810** [comment]
> For ease, I'll re-run the build, which should resolve the credentials issue you're seeing.

Still seeing these issues. I don't understand the rules around forks...

Merging would work, so will just do that 😄 .

**Pull request merged**